<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBManagerClass</class>
 <widget class="QMainWindow" name="DBManagerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>829</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DBManager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>8</number>
      </property>
      <widget class="QWidget" name="mainPage">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="createTagButton">
          <property name="text">
           <string>Добавить Тег</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="createUserButton">
          <property name="text">
           <string>Добавить Пользователя</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="createEventButton">
          <property name="text">
           <string>Добавить Событие</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="createPSiteButton">
          <property name="text">
           <string>Добавить Партнерский сайт</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="createReviewButton">
          <property name="text">
           <string>Добавить Обзор</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="createMediaButton">
          <property name="text">
           <string>Добавить Медиа Файл</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="createPlaceButton">
          <property name="text">
           <string>Добавить Место</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="dbTestButton">
          <property name="text">
           <string>Тест базы и лог</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="createTagWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Добавить тег</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="TagName">
          <property name="text">
           <string>Название тега</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="TagNameEdit"/>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="gobackTag">
             <property name="text">
              <string>Назад</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="createTag">
             <property name="text">
              <string>Создать</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="createUserWidget">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Добавить пользователя</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Name">
          <property name="text">
           <string>Имя</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="NameEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="Surname">
          <property name="text">
           <string>Фамилия</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="SurnameEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="Contacts">
          <property name="text">
           <string>Контакты</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="ContactsEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="Avatar">
          <property name="text">
           <string>Аватар</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="AvatarEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="Login">
          <property name="text">
           <string>Логин</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LoginEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="password">
          <property name="text">
           <string>Пароль</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="PasswordEdit"/>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="gobackUser">
             <property name="text">
              <string>Назад</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="createUser">
             <property name="text">
              <string>Добавить</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="createReviewWidget">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Добавить обзор</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="TextEdit">
          <property name="text">
           <string>Текст</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="Rate">
          <property name="text">
           <string>Оценка</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="RateEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="Whoadded">
          <property name="text">
           <string>Кто добавил</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ListBox" name="WhoAddedEdit"/>
        </item>
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="gobackReview">
             <property name="text">
              <string>Назад</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="createReview">
             <property name="text">
              <string>Добавить</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="createEventWidget">
       <layout class="QFormLayout" name="formLayout">
        <item row="3" column="0">
         <widget class="QLabel" name="ShDescr">
          <property name="text">
           <string>Краткое описание</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="Places">
          <property name="text">
           <string>Места</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLineEdit" name="ShDescrEdit"/>
        </item>
        <item row="4" column="1">
         <widget class="ListBox" name="PlaceEdit"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="FullDescr">
          <property name="text">
           <string>Полное описание</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="Users">
          <property name="text">
           <string>Пользователи</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLineEdit" name="FullDescrEdit"/>
        </item>
        <item row="6" column="1">
         <widget class="ListBox" name="UsersEdit"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="Route">
          <property name="text">
           <string>Маршрут</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLineEdit" name="RouteEdit"/>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="Timetable">
          <property name="text">
           <string>Расписание проведения</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QPlainTextEdit" name="TimetableEdit">
          <property name="plainText">
           <string>Пример (8 марта каждый год):
03/08 00:00:00 - 03/08 23:59:59
*</string>
          </property>
         </widget>
        </item>
        <item row="14" column="0">
         <widget class="QLabel" name="Extra1">
          <property name="text">
           <string>Экстра1</string>
          </property>
         </widget>
        </item>
        <item row="14" column="1">
         <widget class="QLabel" name="Media">
          <property name="text">
           <string>Медиа ресурсы</string>
          </property>
         </widget>
        </item>
        <item row="15" column="0">
         <widget class="QLineEdit" name="Ex1Edit"/>
        </item>
        <item row="15" column="1">
         <widget class="ListBox" name="MediaEdit"/>
        </item>
        <item row="16" column="0">
         <widget class="QLabel" name="Extra2">
          <property name="text">
           <string>Экстра2</string>
          </property>
         </widget>
        </item>
        <item row="16" column="1">
         <widget class="QLabel" name="Rate_2">
          <property name="text">
           <string>Оценка</string>
          </property>
         </widget>
        </item>
        <item row="17" column="0">
         <widget class="QLineEdit" name="Ex2Edit"/>
        </item>
        <item row="17" column="1">
         <widget class="QLineEdit" name="RateEdit_2"/>
        </item>
        <item row="18" column="0" colspan="2">
         <widget class="QWidget" name="widget_4" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="gobackEvent">
             <property name="text">
              <string>Назад</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="createEvent">
             <property name="text">
              <string>Добавить</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="Rev">
          <property name="text">
           <string>Отзывы</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="ListBox" name="RevEdit"/>
        </item>
        <item row="11" column="1">
         <widget class="QLabel" name="PSites">
          <property name="text">
           <string>Сайты партнеров</string>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="ListBox" name="PSiteEdit"/>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Добавить событие</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Имя:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLineEdit" name="nameEdit"/>
        </item>
        <item row="9" column="1">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Тэги</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="ListBox" name="tagBox"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="createPSiteWidget">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Добавить партнерский сайт</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Ref">
          <property name="text">
           <string>Ссылка</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="RefEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="Def">
          <property name="text">
           <string>Описание</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="DefEdit"/>
        </item>
        <item>
         <widget class="QWidget" name="widget_5" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="gobackPSite">
             <property name="text">
              <string>Назад</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="createPSite">
             <property name="text">
              <string>Добавить</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="createMediaWidget">
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_6">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Добавить медиа файл</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="Name_2">
          <property name="text">
           <string>Название ресурса</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLineEdit" name="NameEdit_2"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="Rate_3">
          <property name="text">
           <string>Оценка</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLineEdit" name="RateEdit_3"/>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="Refs">
          <property name="text">
           <string>Ссылки</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPlainTextEdit" name="RefsEdit"/>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QWidget" name="widget_6" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QPushButton" name="gobackMedia">
             <property name="text">
              <string>Назад</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="createMedia">
             <property name="text">
              <string>Добавить</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Описание</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPlainTextEdit" name="plainTextEdit_2"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="createPlaceWidget">
       <layout class="QFormLayout" name="formLayout_3">
        <item row="1" column="0">
         <widget class="QLabel" name="Name_3">
          <property name="text">
           <string>Название места</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="CoordX">
          <property name="text">
           <string>Координата Х</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLineEdit" name="NameEdit_3"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="CoordXEdit"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="Def_2">
          <property name="text">
           <string>Описание</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="CoordY">
          <property name="text">
           <string>Координата У</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="Address">
          <property name="text">
           <string>Адрес места</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0" colspan="2">
         <widget class="QWidget" name="widget_7" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QPushButton" name="gobackPlace">
             <property name="text">
              <string>Назад</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="createPlace">
             <property name="text">
              <string>Добавить</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPlainTextEdit" name="Def_2_Edit"/>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="CoordYEdit"/>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="Media_2">
          <property name="text">
           <string>Спсиок медиаресурсов</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QPlainTextEdit" name="PlaceAddrEdit"/>
        </item>
        <item row="11" column="1">
         <widget class="ListBox" name="MediaEdit_2"/>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_7">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Добавить место</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="dbTestWidget">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QTableWidget" name="tasksTable">
          <column>
           <property name="text">
            <string>Новый столбец</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Объект</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>id</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="logEdit"/>
        </item>
        <item>
         <widget class="QWidget" name="widget_8" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QPushButton" name="gobackDb">
             <property name="text">
              <string>Назад</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="testDb">
             <property name="text">
              <string>Тестировать</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>823</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ListBox</class>
   <extends>QComboBox</extends>
   <header>listbox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="DBManager.qrc"/>
 </resources>
 <connections/>
</ui>
